examples/template_chatml.jinja

```diff
@@ -0,0 +1,3 @@
+{% if not add_generation_prompt is defined %}{% set add_generation_prompt = false %}{% endif %}
```

Comment:
I don't think this logic is correct here. since `add_generation_prompt` is always available within the jinja environment for `apply_chat_template`

Reply:
That was a copy/paste from...I can't recall where, but I have absolutely seen quite a few templates do the same. A google for that string shows how widely it's spread.

Agreed that it seems redundant.

Reply:
Alright, removed.